packages/server/src/script/resendSignupEmails.ts

```diff
@@ -0,0 +1,136 @@
+import * as admin from 'firebase-admin'
+import { FirestoreService } from '../service/firestore'
+import { Letter } from '../service/letter'
+import { RichStateInfo, TwilioResponse } from '../service/types'
+import { sendSignupEmail } from '../service/mg'
+import { storageFileFromId } from '../service/storage'
+import { sendFaxes } from '../service/twilio'
+import { WithId } from '../common'
+import { toContactMethod } from '../common'
+import { toPdfBuffer } from '../service/pdf'
+const { newYorkQuery, previewFields, reason } = process.env.CI ? require('./data/conditions.sample.ts') : require('./data/conditions.nogit.ts')
+
+
+type Query = admin.firestore.Query<admin.firestore.DocumentData>
+const firestoreService = new FirestoreService()
+
+
+/**
+ * Resends emails to all confrimation IDs in list.
+ * @param ids list of confirmation IDs
+ */
+const resendSignups = async (registrations: WithId<RichStateInfo>[], resendReason: string): Promise<void> => {
```

Comment:
A lot of this code repeats code in `trpc.ts`.  Is there a way to share more common code (in a function you factor out)?  Having shared code reduces errors.